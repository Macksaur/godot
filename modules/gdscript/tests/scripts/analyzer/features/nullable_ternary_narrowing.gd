func test():
    var foo: int? = 1
    var result := (foo if foo else 123) + 1 # Should be okay
    print(result)
