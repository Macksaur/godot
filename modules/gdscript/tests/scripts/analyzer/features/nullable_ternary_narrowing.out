GDTEST_OK
2
